react/features/reactions/functions.any.js

```diff
@@ -9,3 +15,74 @@
 export function getReactionsQueue(state: Object) {
     return state['features/reactions'].queue;
 }
+
+/**
+ * Returns reaction key from the reaction message.
+ *
+ * @param {string} message - The reaction message.
+ * @returns {string}
+ */
+export function getReactionKeyByMessage(message: string) {
+    return Object.keys(REACTIONS).filter(key => REACTIONS[key].message === `:${message}:`)[0];
+}
+
+/**
+ * Gets reactions key array from concatenated message.
+ *
+ * @param {string} message - The reaction message.
+ * @returns {Array}
+ */
+export function messageToKeyArray(message: string) {
+    let formattedMessage = message.replace(/::/g, '-');
+
+    formattedMessage = formattedMessage.replace(/:/g, '');
+    const messageArray = formattedMessage.split('-');
+
+    return messageArray.map<string>(reactionMessage => getReactionKeyByMessage(reactionMessage));
```

Comment:
I don't think you need to wrap this mapping into another anonymous function. You could just pass `getReactionKeyByMessage` as it is.